src/app/_variables.scss

```diff
@@ -59,6 +59,21 @@ $easeInCubic: cubic-bezier(0.55, 0.055, 0.675, 0.19);
 $easeOutCubic: cubic-bezier(0.215, 0.61, 0.355, 1);
 $easeInOutCubic: cubic-bezier(0.645, 0.045, 0.355, 1);
 
+// Theme style properties that remain consistent across all themes
+
+// Expose primary brand colour as a css variable so it can be used in themes
+:root {
+  --dim-brand: #{$dim-brand};
+}
```

Comment:
We really cannot have any normal CSS in this file, only SASS variables and mixins.